sdk/@launchdarkly/observability-node/src/client/ObservabilityClient.ts

```diff
@@ -229,18 +237,19 @@ export class ObservabilityClient {
 		)
 		this.processor = new BatchSpanProcessor(exporter, opts)
 
-		this.loggerProvider = new LoggerProvider({
-			resource,
-		})
 		const logsExporter = new SamplingLogExporter(
 			{
 				...config,
 				url: `${config.url}/v1/logs`,
 			},
 			sampler,
 		)
+
 		const logsProcessor = new BatchLogRecordProcessor(logsExporter, opts)
```

Comment:
You can no longer add a log record processor after creating the provider, so this code moved around a little.